packages/apiserver/src/email/email.service.ts

```diff
@@ -8,8 +8,18 @@ const EMAIL_TEMPLATES = {
   RETURN_BOOK_REMINDER: 'return-book-reminder',
 } as const;
 
+const DEFAULT_COVER_IMAGE_PATH = 'assets/MOC-data/BookImage.png';
```

Comment:
Maybe it's better to use EmptyCover.png?
BookImage.png:
![image](https://user-images.githubusercontent.com/48247186/215045784-680d40d1-dd37-4424-aba1-45985101011a.png)
EmptyCover.png:
![image](https://user-images.githubusercontent.com/48247186/215045911-d8d66a0e-76ff-48d9-a181-55037b546aed.png)

By the way, email service can access images in frontapp folder?

Reply:
Changed to EmptyCover.png
It can access